tests/baselines/reference/propertyAccessChain.2.js

```diff
@@ -0,0 +1,16 @@
+//// [propertyAccessChain.2.ts]
+declare const o1: undefined | { b: string };
+o1?.b;
+
+declare const o2: undefined | { b: { c: string } };
+o2?.b.c;
+
+declare const o3: { b: undefined | { c: string } };
+o3.b?.c;
```

Comment:
it would be nice to have a test case with two question-dots in a longer chain of access expressions, just to see what the emit is.